membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -366,20 +377,16 @@ def test_analysis_mean_no_wrap(self, universe):
     # test using dummy Universe with atoms out of boounds
     # with wrap=True (default)
     #   +-----------+          +-----------+
-    #   |   | 7 | 8 | 9        | 9 | 7 | 8 |
+    #   |   | 8 | 9 | 7        | 7 | 8 | 9 |
     #   +-----------+          +-----------+
-    # 4 | 5 | 6 |   |   --->   | 5 | 6 | 4 |
+    # 6 | 4 | 5 |   |   --->   | 4 | 5 | 6 |
     #   +-----------+          +-----------+
-    #   |   | 1 | 2 | 3        | 3 | 1 | 2 |
+    #   |   | 2 | 3 | 1        | 1 | 2 | 3 |
     #   +-----------+          +-----------+
     #
     # test surface in universe with atoms out of bounds in x
-    def test_analysis_get_z_surface_wrap(self, universe_dummy_wrap, dummy_surface):
-        x_bin = y_bin = 3
-        mc = MembraneCurvature(universe_dummy_wrap,
-                               n_x_bins=x_bin,
-                               n_y_bins=y_bin).run()
-        avg_surface = mc.results.average_z_surface
+    def test_analysis_get_z_surface_wrap(self, curvature_unwrapped_universe, dummy_surface):
+        avg_surface = curvature_unwrapped_universe.results.average_z_surface
         assert_almost_equal(avg_surface, dummy_surface)
 
     # test surface in universe with atoms out of bounds in x and y
```

Comment:
You could use the `curvature_unwrapped_universe_xy` fixture here too, right? (the line after)

Reply:
Oh, yeah, I missed that, thanks!